Accompany/Controllers/Core/Home/AddNoteDetailsTableViewController.swift

```diff
@@ -0,0 +1,131 @@
+//
+//  SemesterDetailsTableViewController.swift
+//  Accompany
+//
+//  Created by Kelbin David on 2022-02-10.
+//
+
+import UIKit
+import SnapKit
+
+protocol AddNoteDetailsTableViewControllerDelegate {
+  func add(todo: ToDo)
+  func edit(todo: ToDo)
+}
+
+class AddNoteDetailsTableViewController: UITableViewController {
+  
+  var saveBarButton: UIBarButtonItem = {
+    let barButton = UIBarButtonItem(barButtonSystemItem: .save, target: self, action: #selector(saveButtonTapped))
+    return barButton
+  }()
+  
+  var todo: ToDo?
+  var delegate: AddNoteDetailsTableViewControllerDelegate?
+  var isCompleteButton: UIButton?
+  var toDoTitleTextField: UITextField?
+  var noteTextView: UITextView?
```

Comment:
Please try to divide the properties into different groups, it will be more readable.
```suggestion
  let todoBasicInfoCell = TodoBasicInfoTableViewCell()
  let todoNotesCell = TodoNotesTableViewCell()
  var saveBarButton = UIBarButtonItem(barButtonSystemItem: .save, target: self, action: #selector(saveButtonTapped))
  
  var delegate: TodoFormTableViewControllerDelegate?
  
  var todo: ToDo?
  
```